packages/tpetra/core/src/Tpetra_CrsGraph_def.hpp

```diff
@@ -4805,6 +4828,14 @@ namespace Tpetra {
     const char tfecfFuncName[] = "copyAndPermute: ";
     const bool verbose = verbose_;
 
+    if (true) {
+      const row_graph_type& srcRowGraph =
+        dynamic_cast<const row_graph_type&> (source);
+      copyAndPermuteNew(srcRowGraph, *this, numSameIDs, permuteToLIDs, permuteFromLIDs, INSERT);
```

Comment:
Since `copyAndPermute` calls `copyAndPermuteNew`, would it make sense that `cAPN` simply becomes `cAP`?